source/usage-examples/insertOne.txt

```diff
@@ -0,0 +1,27 @@
+=================
+Create a Document
+=================
+
+.. default-domain:: mongodb
+
+You can create a single document using the `insertOne()
```

Comment:
Somewhat pedantic, but this is more about how to **Insert a Document** or **Add a Document**. The insert methods take a document that you already created and stores it in the collection.

Reply:
Sentence replaced to "
To create a document, define an object variable with the fields and values that you want to specify. You can insert this document using the `insertOne()
<https://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#insertOne>`_
method."